app/assets/javascripts/google_analytics_events.js.coffee

```diff
@@ -0,0 +1,4 @@
+$ ->
+  # Track if user is clicking on stack map pin icon
+  $('.find-it').on 'click', ->
+    ga('send', 'event', 'Stack Map Pin','Clicked')
```

Comment:
I think the we can add three unique values here. We are using Action and Category here, but but we can also add a third Label value. It would be good to add something contextual there that denotes if the action happened on a record page or on a search results page.